dispatch/modules/content/models.py

```diff
@@ -556,7 +557,9 @@ def save_thumbnail(self, image, size, name, label, file_type):
         default_storage.save(name, thumb_file)
 
     def save_tags(self, tag_ids):
-        self.tags.clear()
+
+        # self.tags.clear()
```

Comment:
@psiemens heres the thing i took out that was deleting all my tags, im not sure why my tags arent included in the validated data